src/lidar/device_adapter/rplidar_adapter.py

```diff
@@ -0,0 +1,61 @@
+"""
+Rplidar adapter module.
+"""
+import rplidar
+from math import cos, sin, pi
+import serial.tools.list_ports as port_list
+
+
+class RplidarAdapter:
+
+    def __init__(self):
+        self.__lidar = None
+        self.__seen_polar = []
+        self.__seen_cartesian = []
+        self.init_lidar()
+
+    def init_lidar(self):
+        if self.__lidar is None:
+            try:
+                self.__lidar = rplidar.RPLidar(port_list.comports()[0].device)
```

Comment:
TODO: Utiliser le VID et PID du Lidar pour récupérer son device (en itérant sur comports()).

Reply:
Je suis d'accord :)